csharp/defs.bzl

```diff
@@ -21,7 +21,8 @@ load(
     _import_multiframework_library = "import_multiframework_library",
 )
 load(
-    "//csharp/private:rules/nuget.bzl",
+    "//csharp/private:macros/nuget.bzl",
```

Comment:
I moved this file into macros from the rules folder because... it only has macros. 🤷‍♂